processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -602,6 +604,14 @@ public boolean isBeforeMappingMethod() {
         return Executables.isBeforeMappingMethod( getExecutable() );
     }
 
+    /**
+     * @return returns true for interface methods (see jls 9.4) lacking a default or static modifier and for abstract
+     * methods
+     */
+    public boolean isAbstract() {
+        return executable.getModifiers().contains( Modifier.ABSTRACT );
```

Comment:
This is really nice, I didn't know this about interfaces 😄. We can maybe use the same construct on other places as well.